packages/webapp/src/routes/__tests__/land/add-legal-agreement-parties.spec.js

```diff
@@ -13,205 +47,97 @@ describe(url, () => {
       const response = await submitGetRequest({ url }, 302, {})
       expect(response.headers.location).toEqual(constants.routes.START)
     })
-    it(`should render the ${url.substring(1)} view with undefined other party`, async () => {
-      jest.isolateModules(async () => {
-        let viewResult, contextResult
-        const redisMap = new Map()
-        const legalAgreementDetails = require('../../land/add-legal-agreement-parties')
-        redisMap.set(constants.redisKeys.LEGAL_AGREEMENT_PARTIES, {
-          roles: [
-            {
-              otherPartyName: undefined
-            }
-          ]
-        })
-        const request = {
-          yar: redisMap
-        }
-        const h = {
-          view: (view, context) => {
-            viewResult = view
-            contextResult = context
-          }
-        }
-        await legalAgreementDetails.default[0].handler(request, h)
-        expect(viewResult).toEqual(constants.views.ADD_LEGAL_AGREEMENT_PARTIES)
-        expect(contextResult.partySelectionData.roles[0].otherPartyName).toEqual('')
-      })
-    })
-  })
-  describe('POST', () => {
-    let postOptions
-    beforeEach(() => {
-      postOptions = {
-        url,
-        payload: {}
+    it(`should render the ${url.substring(1)} view with organisation that use wants to change`, async () => {
```

Comment:
Typo - should render the ${url.substring(1)} view with organisation that **user** wants to change